arcsi/api/item.py

```diff
@@ -91,9 +92,20 @@ def add_item():
     # work around ImmutableDict type
     item_metadata = request.form.to_dict()
     # TODO if we could send JSON payloads w/ ajax then this prevalidation isn't needed
+    show_id = int(item_metadata["shows"]) #shortcut in case the show id is needed in the next if statement
     item_metadata["shows"] = [
         {"id": item_metadata["shows"], "name": item_metadata["show_name"]}
     ]
+
+    if item_metadata["number"] == "":
+        show_request = requests.get(app.config["APP_BASE_URL"] + url_for("arcsi.view_show", id=show_id))
```

Comment:
I've read your comment in issue #14 and while it is true it doesn't have perf indication with our current usage we will definitely have to write this over so why not try it right first? This feels bit unnecessary. In my opinion, we have the `Show` object available to instantiate and could query & filter that which returns only the latest episode only if it has one, all in one.